drop-boxes/register-qcml-dropbox/register-qcml-dropbox.py

```diff
@@ -91,11 +91,10 @@ def process(transaction):
 
        	#cegat = False
         f = "source_dropbox.txt"
-       	sourceLabFile = open(os.path.join(incomingPath,f))
-        sourceLab = sourceLabFile.readline().strip()
-        sourceLabFile.close()
-        #if sourceLab == 'dmcegat':
-                #cegat = True
+       	sith open(os.path.join(incomingPath,f)) as sourceLabFile:
```

Comment:
"Always two there are; no more, no less. A master and an apprentice."
```suggestion
       	with open(os.path.join(incomingPath,f)) as sourceLabFile:
```